upup/pkg/fi/cloudup/awstasks/launchtemplate_target_api.go

```diff
@@ -45,6 +45,7 @@ func (t *LaunchTemplate) RenderAWS(c *awsup.AWSAPITarget, a, e, changes *LaunchT
 		MetadataOptions: &ec2.LaunchTemplateInstanceMetadataOptionsRequest{
 			HttpPutResponseHopLimit: t.HTTPPutResponseHopLimit,
 			HttpTokens:              t.HTTPTokens,
+			HttpProtocolIpv6:        aws.String(ec2.LaunchTemplateInstanceMetadataProtocolIpv6Enabled),
```

Comment:
The state should also be read back in Find()

Reply:
I ended up adding it to the model so it can mimic IPv6AddressCount, so it is now in Find().